doc/user/content/sql/system-catalog.md

```diff
@@ -249,6 +249,20 @@ Field          | Type       | Meaning
 `schema_id`    | [`bigint`] | The ID of the schema to which the sink belongs.
 `name`         | [`text`]   | The name of the sink.
 
+### `mz_source_info`
+
+The `mz_source_info` table contains a row for each partition of each source
+in the system.
+
+Field          | Type       | Meaning
+---------------|------------|----------
+`source_name`  | [`text`]   | Materialize's internal name for the source.
+`source_id`    | [`text`]   | Materialize's unique ID for the source. Equivalent to `mz_sources.id`.
+`dataflow_id`  | [`bigint`] | The ID of the dataflow responsible for processing this source.
+`partition_id` | [`text`]   | The ID of the partition within the source. Partition assignments are defined in a source specific manner.
```

Comment:
Thanks, yeah, this seems better. I still think it is funny to mention "assignments" though. Who is assigning the partition, and to what? Can we say something like "The concept of partitions varies by source type. Not all sources types have multiple partitions, in which case there will be only one entry for partition ID `0`."  to avoid the term "assignment"?

Reply:
Yep, your wording makes sense to me -- happy to use that!